modules/buttons/info.html

```diff
@@ -0,0 +1,33 @@
+<style>
+    .info {background-color: #2196F3;} /* Blue */
+    /*.info:hover {background: #0b7dda;}*/
+    button{
+        color: white; 
+        border-radius: 4px;
+        text-align: center;
+        border: none;
+        padding: 14px 28px;
+    }
+    .icon,.text {
+        vertical-align: middle;
+        display: inline-block;
+        align-items: center;
+    }
+    
+</style>
+
+<body>
+    <div>
+        <button class="info">
+         <span class="icon">
+            <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
```

Comment:
I like the fact that you used svg. that's cool and smart.

Reply:
thanks